src/test_driver/linux-cirque/README.md

```diff
@@ -10,6 +10,18 @@ In Project CHIP, cirque is used for integration tests.
 There is a script for running cirque tests, you can find it at
 `scripts/tests/cirque_tests.sh`
 
+## "Docker out of docker" setup
+
+If you don't want cirque break your local environment, you can run
+
+```
+integrations/docker/images/chip-build-cirque/run.sh bash
```

Comment:
this does not work, please consider ./integrations/docker/images/chip-build-cirque/run.sh --privileged --env LOG_DIR=/tmp/cirque_test_output --env CLEANUP_DOCKER_FOR_CI=1 --volume /dev/pts:/dev/pts --volume /tmp/cirque_test_output:/tmp/cirque_test_output -it -- bash

Reply:
Adjusted some arguments for local run.